robot_smach_states/benchmark/benchmark_grasping.py

```diff
@@ -0,0 +1,184 @@
+#! /usr/bin/env python
+
+# System
+import argparse
+import csv
+import time
+
+# ROS
+import rospy
+
+# TU/e Robotics
+from robot_skills.get_robot import get_robot
+
+# Robot Smach States
+import robot_smach_states.util.designators as ds
+from robot_skills.util.entity import Entity
+from robot_skills.util.kdl_conversions import VectorStamped
+from robot_smach_states import Grab, Inspect, ClassificationResult, NavigateToWaypoint, ForceDrive, SetGripper, \
+    Say
+
+try:
+    from typing import List
+except ImportError:
+    pass
+
+def single_item(robot, results_writer, cls, support, waypoint, inspect_from_area=None, non_strict_class=False):
+    grasp_cls = ds.Designator(cls, name='grasp_cls')
+    support_entity = ds.EdEntityDesignator(robot, id=support, name='support_entity')
+    entity_ids = ds.VariableDesignator([], resolve_type=[ClassificationResult], name='entity_ids')
+    waypoint = ds.EdEntityDesignator(robot, id=waypoint, name='waypoint')
+
+    arm = ds.LockingDesignator(ds.UnoccupiedArmDesignator(robot, {}))
```

Comment:
Please add the requirements on the arm, j.i.c. this is reintroduced that it will not immediately break your code.

Reply:
Any suggestions for what that should all be?

Reply:
```suggestion
    arm = ds.LockingDesignator(ds.UnoccupiedArmDesignator(robot, arm_properties={"required_trajectories": ["prepare_grasp"], "required_goals": ["carrying_pose"],
                                                               "required_gripper_types":arms.GripperTypes.GRASPING}))
```

Note that arms should also be imported (from robot_skills import arms)